src/Project/HCA/serialization.content/Content/HCA/Home.yml

```diff
@@ -57,7 +57,7 @@ Languages:
               uid="{7C92BA41-44DD-47AD-A232-65143CBBF3F4}"
               p:before="*"
               s:key="undefined"
-              s:md="/sitecore/layout/Placeholder Settings/Wooli/Project/Wooli/wooli-content" />
+              s:md="/sitecore/layout/Placeholder Settings/HCA/Project/HCA/wooli-content" />
```

Comment:
Did you rename placeholder setting `wooli-content`?

Reply:
yes, later I'll change this one and host name

Reply:
yes, later I'll change this one and host name